README.md

```diff
@@ -20,17 +20,52 @@ kubectl apply -f \
   https://github.com/cert-manager/cert-manager/releases/download/v1.8.2/cert-manager.yaml
 ```
 
-2. The Bottlerocket update operator can then be installed using the recommended configuration defined in [bottlerocket-update-operator.yaml](https://github.com/bottlerocket-os/bottlerocket-update-operator/blob/1.0.0/yamlgen/deploy/bottlerocket-update-operator.yaml).
-This YAML file can also be found in the [Brupop release artifacts](https://github.com/bottlerocket-os/bottlerocket-update-operator/releases):
+Or if you're using helm:
 
 ```sh
-kubectl apply -f ./bottlerocket-update-operator.yaml
+# Add the cert-manager helm chart
+helm repo add jetstack https://charts.jetstack.io
+
+# Update your local chart cache with the latest
+helm repo update
+
+# Install the cert-manager (including its CRDs)
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.8.2 \
+  --set installCRDs=true
 ```
 
-This will create the required namespace, custom resource definition, roles, deployments, etc., and use the latest update operator image available in [Amazon ECR Public](https://gallery.ecr.aws/bottlerocket/bottlerocket-update-operator).
+2. The Bottlerocket update operator can then be installed using helm:
 
-Note: The above link points to the latest version of the Update Operator, `v1.0.0`.
-Be sure to use the git tag for the Update Operator release that you plan to deploy.
+```sh
+# Add the bottlerocket-update-operator chart
+helm repo add brupop https://bottlerocket-os.github.io/bottlerocket-update-operator
+
+# Update your local chart cache with the latest updates
+helm repo update
+
+# Install the brupop CRD
+helm install \
+  brupop-crd bottlerocket-update-operator/bottlerocket-shadow \
+  --create-namespace \
+  --version v1.0.0 \
+
+# Install the brupop operator
+helm install \
+  brupop bottlerocket-update-operator/bottlerocket-shadow \
+  --version v1.0.0
+```
```

Comment:
Good question!

This is a common Helm convention: the CRD chart is separate from the consumer of the CRD. This is because helm will reconcile resources from a operator chart in the most efficient way possible:

> Helm is optimized to load as many resources into Kubernetes as fast as possible. By design, Kubernetes can take an entire set of manifests and bring them all online (this is called the reconciliation loop).

> But there's a difference with CRDs.

> For a CRD, the declaration must be registered before any resources of that CRDs kind(s) can be used. And the registration process sometimes takes a few seconds.

[Helm Docs - Install a CRD Declaration Before Using the Resource](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#install-a-crd-declaration-before-using-the-resource)

There _is an option_ in Helm to let the `helm install` handle this for you. However, this has the follow caveats:

> There is no support at this time for upgrading or deleting CRDs using Helm. This was an explicit decision after much community discussion due to the danger for unintentional data loss. Furthermore, there is currently no community consensus around how to handle CRDs and their lifecycle. As this evolves, Helm will add support for those use cases.

[Helm docs - Some caveats (and explanations)](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations)

Instead, I seperated the two charts to make handling the bottlerocket shadows explicit:
- Cluster operators can can manage the `brs` CRD out of band from the operator itself
  - We avoid the problem of not being able to upgrade/downgrade CRD versions (even though brs v2 is relatively stable)

There is also an argument that has been made in the community that cluster custom resource definitions should only be installable by a cluster admin (where strong RBAC rules are often enforced) but other users may have access to view and edit the `brs` resources and namespace.

In short, this was a deliberate decision to enforce stronger RBAC practices for our users and gets around some issues with versioning CRDs in helm. However, I'm not opposed to exploring a single install option.